src/components/Balance.tsx

```diff
@@ -16,7 +16,15 @@ interface BalanceProps extends TextProps {
 }
 
 const StyledText = styled(Text)<TextProps>`
-  color: ${({ isDisabled, color, theme }) => (isDisabled ? theme.colors.textDisabled : color)};
+  color: ${({ isDisabled, color, theme }) => {
+    if (isDisabled) {
+      return theme.colors.textDisabled
+    }
+    if (color !== Balance.defaultProps.color) {
+      return color
+    }
+    return theme.colors.text
+  }};
```

Comment:
color: ${({ isDisabled, color, theme }) => isDisabled ? theme.colors.textDisabled : color !== Balance.defaultProps.color ? color : theme.color.text}};

Reply:
I think nested ternary conditions is not readable simply

Reply:
Nested ternaries raise an error with our eslint config